### ISSUE TITLE
mod doesnt show in modmenu or on title screen

### ISSUE THREAD

**Issue by shamshitty** [ReplayMod/ReplayMod#712]
I've just downloaded the iris compatible version of this mod so I can actually render stuff with sodium installed, but it doesn't appear in the moment or on the title screen anywhere like the version that _doesnt_ support sodium does.

is there any way to fix this or do I have to uninstall all my mods and install optifine as hardly any of my mods work with optifine installed?

**Comment by Kepler-17c** [comment]
All versions are compatible with Iris - you probably mean our custom Sodium version:  
Because of Sodium's changes in chunk loading, rendering doesn't work with the official mod. Our developer has proposed a fix, that is pending for the official Sodium and provides adjusted versions on our download page.

TL;DR: You need both, ReplayMod and our custom Sodium. There is no bundle or compatible ReplayMod.

Also, please use our official support Discord https://discord.gg/5GR7RSb for future questions.

**Comment by shamshitty** [comment]
ohh ok I understand, thank you
